### PR TITLE
Add nlp

### PR DESCRIPTION
**Pull request by melvinthemok**

### CODE REVIEW

File: utils.js [hunk 1]
```diff
@@ -1,3 +1,27 @@
+const speakeasy = require('speakeasy-nlp')
+
+const { POSITIVE, NEGATIVE, CONFUSED, NEUTRAL } = require('./responses')
+
+const determineEmotionOf = (answer) => {
+
+  const positivity = speakeasy.sentiment.positivity(answer).score
+  const negativity = speakeasy.sentiment.negativity(answer).score
+
+  switch (true) {
```
Comment (yuanzai) [inline utils.js]: is this a js thing? switching on true? seems haxxx.

File: index.js [hunk 2]
```diff
@@ -1,22 +1,38 @@
...
-rl.on('line', () => {
-  alternator
-  ? console.log(randomlySelectOneOfThe(SWEET_NOTHINGS))
-  : console.log(randomlySelectOneOfThe(ASSERTIONS))
-  alternator = !alternator
+rl.on('line', (answer) => {
+  switch (determineEmotionOf(answer)) {
+    case POSITIVE:
+      console.log(randomlySelectOneOfThe(INSTRUMENTS_OF_TOUGH_LOVE))
+      clearSignalFromLastAnswer = true
```
Comment (yuanzai) [inline index.js]: how functional are js lambda? changing state outside of the lambda seems questionable?

Comment (melvinthemok) [reply index.js]: https://nodejs.org/api/readline.html

File: utils.js (hunk 1 above)
Comment (melvinthemok) [reply utils.js]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch#Rewriting_multiple_If_statements_with_Switch

**Pull request merged**